hyperspy/drawing/utils.py

```diff
@@ -724,10 +749,6 @@ def plot_images(images,
     ax_im_list = [0] * len(isrgb)
     # Loop through each image, adding subplot for each one
     for i, ims in enumerate(images):
-        # Get handles for the signal axes and axes_manager
-        axes_manager = ims.axes_manager
```

Comment:
I removed these few lines, because I think these are doing nothing! If case I am missing something, please yell at me and I will put it back! ;)